frontend/src/components/schedules/ListView.vue

```diff
@@ -306,7 +307,8 @@ export default {
 
       api
         .updateScheduleName(this.schedule[index].id, cleanedName)
-        .then(() => this.schedule[index].name = cleanedName)
+        .then(() => this.scheduleLocal[index].name = cleanedName) // steven: put schedule as a local variable in data (line 129) to avoid mutation error, not sure if right move
```

Comment:
What was the error you were getting?

Reply:
This particular one is the 'no-mutating-props' error (https://eslint.vuejs.org/rules/no-mutating-props.html) and this occurs when we try to set a value to a prop in a component such as 'this.prop = value'. So I tried storing the prop as a local data variable (scheduleLocal), which got rid of the error.

I fixed the 'multi-word-component-names' error by just turning it off, and I fixed the "no-prototype-builtins" error by calling the method from Object.prototype as stated in https://eslint.org/docs/latest/rules/no-prototype-builtins